src/jvm/main/org/jetbrains/kotlinx/lincheck/strategy/managed/LoopDetector.kt

```diff
@@ -914,13 +914,13 @@ internal fun afterSpinCycleTraceCollected(
         var firstI = spinCycleFirstTracePointCallStackTrace.lastIndex
         var count = 0
         while (firstI >= 0) {
-            val identifier = spinCycleFirstTracePointCallStackTrace[firstI].methodId
+            val identifier = spinCycleFirstTracePointCallStackTrace[firstI].methodInvocationId
             // Comparing corresponding calls.
-            if (identifier != currentCallStackTrace[currentI].methodId) break
+            if (identifier != currentCallStackTrace[currentI].methodInvocationId) break
```

Comment:
`methodInvocation` or `methodCall`?

Reply:
I would prefer `methodInvocationId` or `methodCallId` name. 

The reason is that `methodInvocationId` is just an `Int` -- it is an identifier of method call, not a method call itself. 
The identifier is computed as hash-code of the method id and the hash-codes of all its arguments.

In case of `methodInvocation` name, I would expect the field to store the information about the method call itself, that is method name and all its arguments (as objects, not as their hash-codes).